boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -105,7 +106,7 @@ hardware features:
 | WDT       | on-chip    | watchdog             |
 +-----------+------------+----------------------+
 
-The nrf5340pdk_nrf5340_cpunet board configuration supports the following
+The nrf5340dk_nrf5340_cpunet board configuration supports the following
 hardware features:
 
 +-----------+------------+----------------------+
```

Comment:
The table below should be corrected. ADC and PWM are not available in the Network core.

Reply:
Thanks, not sure why this was wrong anyways,
Updated now.